File: httpsnippet/src/test/resources/output/python/python3/jsonObj-null-value.py
import http.client

conn = http.client.HTTPConnection("mockbin.com", "80")

payload = "{\"foo\":null}"

headers = {
	"content-type": "application/json"
}

conn.request("POST", "/har", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: httpsnippet/src/test/resources/output/node/request/jsonObj-null-value.js
var request = require("request");

var options = {"headers":{"content-type":"application/json"},"method":"POST","json":true,"body":{"foo":null},"url":"http://mockbin.com/har"};

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});
